src/near-rpc-provider.spec.ts

```diff
@@ -71,6 +71,13 @@ describe('NearRpcProvider', () => {
   //   })
   // })
 
+  describe('getCode', () => {
+    it('should get the contract code', async () => {
```

Comment:
Could you please add test for get the code by block id and block hash too?

Reply:
On it!